FunctionSignatureMap.php

```diff
@@ -8555,14 +8559,15 @@
 'oci_cancel' => ['bool', 'stmt'=>'resource'],
 'oci_client_version' => ['string'],
 'oci_close' => ['bool', 'connection'=>'resource'],
-'OCI_Collection::append' => ['bool', 'value'=>'mixed'],
-'OCI_Collection::assign' => ['bool', 'from'=>'OCI_Collection'],
-'OCI_Collection::assignelem' => ['bool', 'index'=>'int', 'value'=>'mixed'],
-'OCI_Collection::free' => ['bool'],
-'OCI_Collection::getelem' => ['mixed', 'index'=>'int'],
-'OCI_Collection::max' => ['int'],
-'OCI_Collection::size' => ['int'],
-'OCI_Collection::trim' => ['bool', 'num'=>'int'],
+'OCI-Collection::append' => ['bool', 'value'=>'mixed'],
```

Comment:
I might just remove OCI-Collection's signatures from Phan entirely, because it can't analyze them. (or change everything added elsewhere from OCI-Lob to OCI_Lob

It assumes internal modules declares classes with valid FQSENs, which is true for everything except the `oci` module.

https://github.com/phan/phan/issues/2222#issuecomment-445508988

Reply:
Yeah I don't think Psalm can analyse them either (but nobody's ever asked, and I'm hoping nobody ever does)

Reply:
And it seems like it'd cause a crash in Psalm as well

It might be analyzable with the correct stub files for the underscore version (but users might be mislead about what the class name is).

```php
            if (stripos($return_type, 'oci-') !== false) {
                return;
            }
```

Reply:
Now it doesn’t

Reply:
Added a comment suggesting how to migrate PHP away from the name oci-collection in https://bugs.php.net/bug.php?id=58319